share/setup.nsi.in

```diff
@@ -2,6 +2,11 @@ Name "@PACKAGE_NAME@ (64-bit)"
 
 RequestExecutionLevel highest
 SetCompressor /SOLID lzma
+SetDateSave off
```

Comment:
👍 [`SetDateSave`](https://nsis.sourceforge.io/Reference/SetDateSave). I think adding the debugging info below is fine as well.